README.md

```diff
@@ -18,18 +23,47 @@ As a quantum circuit compiler, it is fully aware of the semantics of each gate a
 It understands the quantum programming language cQASM 3 and will support additional quantum programming languages in the future.
 It is developed in modern Python and follows best practices.
 
+## User manual
+
+OpenSquirrel documentation is hosted through GitHub Pages [here](https://QuTech-Delft.github.io/OpenSquirrel/).
+
+## File organization
+
+For development, see:
+
+- `opensquirrel`: source files.
+- `test`: test files.
 
-## Installation
+For build process, continuous integration, and documentation:
+
+- `.github`: GitHub Actions configuration files.
+- `docs`: documentation files.
+- `scripts`: documentation helper script.
+
+## Dependencies
+
+All dependencies are managed via `poetry`.
```

Comment:
```suggestion
All dependencies are managed via [Poetry](https://python-poetry.org/).
```

Reply:
I'm leaving it like: `All dependencies are managed via [poetry](https://python-poetry.org/).`.